leveldbjni/src/main/native-package/autotools/ltmain.sh

```diff
@@ -1,9 +1,12 @@
+#! /bin/sh
+## DO NOT EDIT - This file generated from ./build-aux/ltmain.in
+##               by inline-source v2014-01-03.01
 
-# libtool (GNU libtool) 2.4.2
+# libtool (GNU libtool) 2.4.6
```

Comment:
I don't think we need to check in this code change since this is automatically generated?

Reply:
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836